website/docs/d/cloud_project_database_prometheus.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory : "Managed Databases"
+---
+
+# ovh_cloud_project_database_prometheus (Data Source)
+
+Use this data source to get information about a prometheus of a database cluster associated with a public cloud project.
```

Comment:
optional, but maybe you could add something like `for mongodb, please use the other datasource ...`